io.github.mfat.tvhplayer.yml

```
@@ -0,0 +1,101 @@
+app-id: io.github.mfat.tvhplayer
+runtime: org.kde.Platform
+runtime-version: '5.15-24.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: '5.15-24.08'
+command: tvhplayer
+separate-locales: false
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+  - rm -f /var/run/user/*/wayland*
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
```

```suggestion
  - --socket=x11
```

Given what you wrote your application doesn't support wayland.

---

I used sockets=x11 and the app runs fine under wayland in Fedora.